projects/driver_controls/test/test_pedal_calibration_test.c

```diff
@@ -0,0 +1,139 @@
+#include "ads1015_def.h"
+#include "debouncer.h"
+#include "delay.h"
+#include "gpio_it.h"
+#include "input_event.h"
+#include "interrupt.h"
+#include "log.h"
+#include "pedal_calibration.h"
+#include "pedal_calibration_fsm.h"
+#include "persist.h"
+#include "string.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+static PedalCalibrationStorage s_storage;
+static Ads1015Storage s_ads1015_storage;
+static ThrottleCalibrationData s_throttle_calibration_data;
+
+// Inputs user might want to change:
+#define TEST_PEDAL_CALIBRATION_ADC_CHANNEL_A ADS1015_CHANNEL_0
+#define TEST_PEDAL_CALIBRATION_ADC_CHANNEL_B ADS1015_CHANNEL_1
+#define TEST_PEDAL_CALIBRATION_BRAKE_PERCENTAGE 30
+#define TEST_PEDAL_CALIBRATION_COAST_PERCENTAGE 30
+#define TEST_PEDAL_CALIBRATION_SAFETY_FACTOR 2
```

Comment:
What is this a safety factor for?

Reply:
Dont we need a safety factor? Like you mentioned once there might be humidity or smth that would change up the numbers a little bit. So this technically lets us have a bigger tolerance.

Reply:
I'm not asking why we have a safety factor, it's a question of what this is supposed to be a factor for.